Documentation/project-docs/clr-configuration-knobs.md

```diff
@@ -1,829 +1,1020 @@
-System.Threading.ThreadPool.MinThreads|Override MinThreads for the ThreadPool worker pool|numeric
-System.Threading.ThreadPool.MaxThreads|Override MaxThreads for the ThreadPool worker pool|numeric
+`System.GC.Concurrent` | Enable concurrent GC | boolean
+`System.GC.Server` | Enable server GC | boolean
+`System.GC.RetainVM` | Put segments that should be deleted on a standby list for future use instead of releasing them back to the OS | boolean
+`System.Threading.ThreadPool.MinThreads` | Override MinThreads for the ThreadPool worker pool | numeric
+`System.Threading.ThreadPool.MaxThreads` | Override MaxThreads for the ThreadPool worker pool | numeric
+
+
 
 ## Environment/Registry Configuration Knobs
 
-This table is machine-generated from commit 456b418 on 08/22/17. It might be out of date.
+This table was machine-generated from commit [5f0c8040](https://github.com/dotnet/coreclr/commit/5f0c80406a370c5d6b41a379e015511cdaf07dda) on 2018-01-15. It might be out of date.
 
 When using these configurations from environment variables, the variables need to have the `COMPlus_` prefix in their names. e.g. To set DumpJittedMethods to 1, add the environment variable `COMPlus_DumpJittedMethods=1`.
 
 See also [Setting configuration variables](../building/viewing-jit-dumps.md#setting-configuration-variables) for more information.
 
+#### Tables
+1. [AppDomain Configuration Knobs](#appdomain-configuration-knobs)
+2. [ARM Configuration Knobs](#arm-configuration-knobs)
+3. [Assembly Loader Configuration Knobs](#assembly-loader-configuration-knobs)
+4. [Conditional breakpoints Configuration Knobs](#conditional-breakpoints-configuration-knobs)
+5. [CSE Configuration Knobs](#cse-configuration-knobs)
+6. [Debugger Configuration Knobs](#debugger-configuration-knobs)
+7. [Diagnostics (internal general-purpose) Configuration Knobs](#diagnostics-internal-general-purpose-configuration-knobs)
+8. [Exception Handling Configuration Knobs](#exception-handling-configuration-knobs)
+9. [Garbage collector Configuration Knobs](#garbage-collector-configuration-knobs)
+10. [GDBJIT Configuration Knobs](#gdbjit-configuration-knobs)
+11. [IBC Configuration Knobs](#ibc-configuration-knobs)
+12. [Interop Configuration Knobs](#interop-configuration-knobs)
+13. [Interpreter Configuration Knobs](#interpreter-configuration-knobs)
+14. [JIT Configuration Knobs](#jit-configuration-knobs)
+15. [JIT Hardware Intrinsics Configuration Knobs](#jit-hardware-intrinsics-configuration-knobs)
+16. [Jit Pitching Configuration Knobs](#jit-pitching-configuration-knobs)
+17. [JIT64 Configuration Knobs](#jit64-configuration-knobs)
+18. [Loader Configuration Knobs](#loader-configuration-knobs)
+19. [Loader heap Configuration Knobs](#loader-heap-configuration-knobs)
+20. [Log Configuration Knobs](#log-configuration-knobs)
+21. [MDA ? Configuration Knobs](#mda--configuration-knobs)
```

Comment:
Perhaps this category may have more self documenting name instead of `MDA ?`

Reply:
Everything in this category can be deleted. All knobs in this category should be unreachable. We do not have these in CoreCLR.

(MDA stands for "Managed debugging assistant".)

Reply:
> MDA stands for "Managed debugging assistant"

@4creators I was just about to say that you could use the [glossary](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/glossary.md) to look up acronyms like this, but it isn't in there!!

@jkotas Shall I send a PR to add `MDA`, or is it too niche?

Reply:
> @jkotas Shall I send a PR to add `MDA`

Sounds good to me. Thanks!